atom/browser/atom_browser_client.cc

```diff
@@ -791,15 +791,16 @@ void AtomBrowserClient::RenderProcessExited(
   }
 }
 
+void OnOpenExternalFinished(const std::string& msg) {
+  LOG(INFO) << msg;
```

Comment:
```suggestion
```

Reply:
should it just be empty then do you think?

Reply:
if it does nothing, why have it?

Reply:
I think it was intended to adjust to the openExternal api that needed a callback as argument. @codebytere you can use `base::NullCallback()` in such cases.

Reply:
yep! thanks :D